### DISCUSSION TITLE
Sheet for playing Sorcerers?

### DISCUSSION THREAD

**Discussion by Beartoya** [JohanFalt/Foundry_WoD20#924] [Q&A] [answered]
Is there a way to play Sorcerers from [W20: Sorcerer ](https://www.drivethrurpg.com/en/product/401413/m20-sorcerer)? I tried setting them up in Mortals, but you can't have Quintessence on the sheet, from what I can tell anyway, and in mage there isn't a sheet for Paths, Rituals, or Psychic Phenomena. Thank you for your time! They also play a little differently, most importantly, not being bound by Paradox.  

[For reference](https://mrgone.rocksolidshells.com/pdf/M20/M20_Sorcerer_2-Page_Interactive.pdf)

**Comment by JohanFalt** [comment, marked as answer]
They will be available in v4.2 (next version).

**Comment by JohanFalt** [comment]
![{4D838E31-8218-43A1-9B26-B2B305022341}](https://github.com/user-attachments/assets/aea3dd00-4226-4a54-b0cf-d967e5af4da5)

![{3C9DAC44-9DC2-4AF1-8F1F-3C12DB567851}](https://github.com/user-attachments/assets/378c7153-4540-4939-84d8-663fbf994649)

![{E806D3E8-85A7-4BE0-9865-0E0E8F874D66}](https://github.com/user-attachments/assets/4acb2275-ef85-4b4f-a89b-6efbb7d3e077)
